Lib/test/test_clinic.py

```diff
@@ -1152,6 +1244,20 @@ def test_left_group(self):
                 Attributes for the character.
         """)
 
+    def test_unmatched_right_square_paren(self):
+        err = "Function 'addch' has a ']' without a matching '['."
```

Comment:
I thought we had a test for this already?

Reply:
You're right. Not sure what went on there. Maybe a merge race, maybe a lack of caffeine :)

I'll remove this test.